xml/mysql_banners.xml

```diff
@@ -998,6 +998,15 @@
     <param pos="0" name="service.product" value="MariaDB"/>
     <param pos="0" name="service.edition" value="Enterprise Edition"/>
   </fingerprint>
+  <fingerprint pattern="(\d{1,2}\.\d{1,2}\.\d{1,2})-maxscale">
```

Comment:
We should consider adding the regex for the initial versions and anchoring on the full field if we know it is always in the format.